src/java.desktop/share/classes/java/awt/Window.java

```diff
@@ -241,7 +241,12 @@
     private transient Component temporaryLostComponent;
 
     static boolean systemSyncLWRequests = false;
-    boolean     syncLWRequests = false;
+
+    /**
+     * Focus transfers for the lightweight components request should be made
+     * synchronously.
+     */
```

Comment:
More simpler way "Focus transfers should be synchronous for lightweight component requests"